tools/config-markdown.py

```
@@ -1,207 +1,159 @@
             for sub_item in item['sub']:
-                file_name_sub_technical = f"{sub_item['id']}.technical.md"
-                file_path_sub_technical = os.path.join(item_dir, file_name_sub_technical)
-                markdown_content_sub_technical = create_markdown_technical(sub_item)
-                sub_anchor_links = generate_anchor_links(sub_item)
-                sub_anchor_links_content = "\n".join(sub_anchor_links) + "\n\n"
-                full_content_sub_technical = sub_anchor_links_content + markdown_content_sub_technical
-
-                with open(file_path_sub_technical, 'w') as f:
-                    f.write(full_content_sub_technical)
+                sub_anchors = "\n".join(generate_anchor_links(sub_item)) + "\n\n"
+                sub_technical_md = create_markdown_technical(sub_item)
```

Ruff (0.8.2)

Undefined name `create_markdown_technical`

(F821)